pkg/client/cli/telepresence_test.go

```diff
@@ -428,21 +430,33 @@ func (cs *connectedSuite) SetupSuite() {
 
 	// Connect + quit before we change contexts to ensure the
 	// traffic-manager is installed
-	_, stderr := telepresence(cs.T(), "connect")
+	configDir := cs.T().TempDir()
+	registry := dtest.DockerRegistry(c)
+	configYml := fmt.Sprintf(`
+logLevels:
+  rootDaemon: debug
+images:
+  registry: %s
+timeouts:
```

Comment:
I don't think a merge blocker, but if we need these defaults for a bunch of tests, I wonder if our defaults are wrong and we should bump them to these values. imo, the tests should run with as many default settings as possible, only changing them if we are specifically testing that config option.